libbcachefs.c

```diff
@@ -299,9 +299,9 @@ struct bch_sb *bch2_format(struct bch_opt_strs	fs_opt_strs,
 
 		if (i->sb_offset == BCH_SB_SECTOR) {
 			/* Zero start of disk */
-			static const char zeroes[BCH_SB_SECTOR << 9];
+			static const char zeroes[BCH_SB_SECTOR << 13];
 
-			xpwrite(i->fd, zeroes, BCH_SB_SECTOR << 9, 0);
+			xpwrite(i->fd, zeroes, BCH_SB_SECTOR << 13, 0);
```

Comment:
Instead of doing it this way it'd probably be more reliable to do whatever wipefs is doing - btrfs stores a superblock at the _end_ of the partition as well.

it looks like from scanning the util-linux source it's calling blkid_do_wipe(), so that shouldn't be too hard.

Reply:
fixed in commit bdf7380cf119362b96657d18f426105495aac9d1